io/snmp/snmp.js

```diff
@@ -65,7 +65,18 @@ module.exports = function (RED) {
         this.on("input", function (msg) {
             var host = node.host || msg.host;
             var community = node.community || msg.community;
-            var varbinds = (node.varbinds) ? JSON.parse(node.varbinds) : msg.varbinds;
+            var varbinds;
+            if (node.varbinds) {
+                try { 
+                    varbinds=JSON.parse(node.varbinds);
+                }
+                catch (e) {
+                    node.error("Error parsing varbinds ");
```

Comment:
`node.error("Error parsing varbinds ");` won't be raised to the catch block without adding the `msg` parameter: `node.error("Error parsing varbinds", msg);`